### ISSUE TITLE
echo finds the pimatic devices no longer

### ISSUE THREAD

**Issue by baddle** [michbeck100/pimatic-echo#53]
Hello,

I use 4 Echo (2. generation) with pimatic (latest version), the pimatic plugin (0.5.2), some yeelight, homeduino and TP-Link smartplug devices. Since some days the echos no longer find new devices but the older ones still worked. After I noticed that I restarted the pimatic-pi but the problem was still there. Even a reinstall of the echo-plugin and deleting all old devices brought no success. Now the Echos find my devices anymore (discover devices was activated).

There are no errors in the log and i can access the api under <pimatic-ip>/api/<hex-string>/lights/. 

Did I miss something? I would be glad about help.

baddle

**Comment by baddle** [comment]
Update:
I replaced my Echo Dot (2. generation) some weeks ago with an echo (2. generation). I believe since then my echos find new devices anymore.

I have added a borrowed echo dot (2. generation) and the dot found all my devices but after I disconnect the dot again all devices are gone.

Are Echo (2. generation) not supported? I read the readme and I understood that only the port 80 must be available for an Echo 2nd gen. Did I miss something?

I attached the description.xml, the lights.json and the dicovery mode log.
[description.txt](https://github.com/michbeck100/pimatic-echo/files/1608686/description.txt)
[lights.txt](https://github.com/michbeck100/pimatic-echo/files/1608687/lights.txt)
[log.txt](https://github.com/michbeck100/pimatic-echo/files/1608688/log.txt)
